src/index.js

```diff
@@ -172,7 +172,9 @@ class IOpipeWrapperClass {
     });
   }
   fail(err) {
-    this.sendReport(err, this.originalContext.fail);
+    this.sendReport(err, () => {
```

Comment:
No, that'd be a reassignment which we don't want. It would technically work but will throw an eslint error (rightfully).